package.json

```
@@ -50,6 +48,8 @@
   },
   "peerDependencies": {
     "expo": ">=47.0.0",
+    "react-native": "*",
+    "react": "*",
```

I don't think this is the best solution, this would allow to install any version of react-native and react.
I'd suggest setting specific versions, and allowing compatibility version updates:
```suggestion
    "react-native": "^0.81.1",
    "react": "^19.1.1",
```